frontend/src/layouts/NetworkDropdown.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <q-btn-dropdown
+    color="primary"
+    no-caps
```

Comment:
Perhaps make this button a little more subtle
```suggestion
    color="primary"
    no-caps
    outline
    rounded
```

Reply:
Or if you switch to the `BaseSelect` component `:outline="true"` and `:rounded="true"`, but it seems we'd have to add a prop for `rounded` as it's not currently in the `BaseSelect` component